files/mods.json

```diff
@@ -2690,5 +2690,37 @@
         "description": "this is purely internal",
         "hidden": true,
         "file": "no"
+    },
+    {
+        "id": "secretroutesmod",
```

Comment:
Putting SRM will put you at the bottom of the skyclient mod list. Put yourself somewhere with the other dungeons mods, I would say right above DRM and DG.